ksql-udf-quickstart/src/main/resources/archetype-resources/src/test/java/SummaryStatsUdafTests.java

```diff
@@ -0,0 +1,99 @@
+/*
+ * Copyright 2019 Confluent Inc.
+ *
+ * Licensed under the Confluent Community License (the "License"); you may not use
+ * this file except in compliance with the License.  You may obtain a copy of the
+ * License at
+ *
+ * http://www.confluent.io/confluent-community-license
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OF ANY KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+
+package ${package};
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.params.provider.Arguments.arguments;
+
+import io.confluent.ksql.function.udaf.Udaf;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.Arguments;
+import org.junit.jupiter.params.provider.MethodSource;
+
+import java.util.HashMap;
+import java.util.Map;
+import java.util.stream.Stream;
+
+/**
+ * Example class that demonstrates how to unit test UDAFs.
+ */
+public class SummaryStatsUdafTests {
+
+  @Test
+  void mergeAggregates() {
+    final Udaf<Double, Map<String, Double>> udaf = SummaryStatsUdaf.createUdaf();
+    final Map<String, Double> mergedAggregate = udaf.merge(
+      // (sample_size, sum, mean)
+      aggregate(3.0, 3300.0, 1100.0),
```

Comment:
nit: the math on the mean doesn't actually add up on this test, cos the mean on the two source aggregates is ignored.

i.e.

```
((3.0 * 1100.0) + (7.0 * 800.0)) / (3.0 + 7.0) != 1000.0
```

This is mainly because:

```
aggregate(7.0, 6700.0, 800.0)
```

Has the wrong mean.  Easiest fix... drop the `mean` from this `aggregate` call and compute it internally on the function, or maybe set it to something like a constant called `IGNORED`...

Reply:
oops, nice catch @big-andy-coates. I went with your last suggestion and fixed the mean value in this line :)